web/cobrands/sass/_base.scss

```diff
@@ -712,15 +712,23 @@ ul.error {
 }
 
 // this is a skip to nav for mobile users only
+label[for="main-nav-btn"] {
+  font-weight: normal;
+  margin: 0;
+}
+
 #nav-link {
   width: 3em;
   height: 0;
   padding-top: 3em;
   overflow: hidden;
-  background: transparent url(../fixmystreet/images/#{$menu-image}.png) center center no-repeat;
```

Comment:
If we’re not using the `menu-*.png` any more, should we `git rm` them from `/web/cobrands/fixmystreet/images/`?

Reply:
I’ve added a commit that removes the `menu-*.png` files.